packages/multichain/src/scope/constants.ts

```diff
@@ -0,0 +1,56 @@
+import MetaMaskOpenRPCDocument from '@metamask/api-specs';
+
+import type { NonWalletKnownCaipNamespace } from './types';
+
+/**
+ * ScopeStrings for offchain methods that are not specific to a chainId but are specific to a CAIP namespace.
+ */
+export enum KnownWalletScopeString {
+  Eip155 = 'wallet:eip155',
+}
+
+/**
+ * Methods that do not belong exclusively to any CAIP namespace.
+ */
+export const KnownWalletRpcMethods: string[] = [
+  'wallet_registerOnboarding',
+  'wallet_scanQRCode',
+];
+
+const WalletEip155Methods = ['wallet_addEthereumChain'];
+
+/**
+ * All MetaMask methods, except for ones we have specified in the constants above.
+ */
+const Eip155Methods = MetaMaskOpenRPCDocument.methods
```

Comment:
Nit: If we add more methods to our OpenRPC document without updating this file, it would result in them being assumed as EIP-155 methods even though they may not be. This seems like a foot-gun.

Maybe we could reduce the risk of this by filtering for methods that start with an EIP-155 related prefix? Or we could add a snapshot test so that new methods don't get added without us noticing.

Reply:
done here: https://github.com/MetaMask/core/pull/4784/commits/67bd1d2db957afed69229c6ed26dfdeaa2d258e6